pytest_astropy_header/display.py

```diff
@@ -11,14 +11,10 @@
 import locale
 import os
 import sys
-from collections import OrderedDict
 
 TESTED_VERSIONS = {}
-PYTEST_HEADER_MODULES = OrderedDict([('Numpy', 'numpy'),
-                                    ('Scipy', 'scipy'),
-                                    ('Matplotlib', 'matplotlib'),
-                                    ('h5py', 'h5py'),
-                                    ('Pandas', 'pandas')])
+DEFAULT_PACKAGES = ("Numpy", "Scipy", "Matplotlib", "h5py", "Pandas")
+PYTEST_HEADER_MODULES = {pkg: pkg.lower() for pkg in DEFAULT_PACKAGES}
```

Comment:
I don't think lowercase assumption is always true but we are also unlikely to add more package to this list, so I guess it is ok.

Reply:
The lowercase assumption is true at the moment, and if more default packages are added then these lines will have to be edited anyways.